templates/service-template.yml

```diff
@@ -285,6 +285,36 @@ parameters:
   description: A list of users that are allowed to create standard KAFKA instances via their organisation in a yaml format.
   value: "[{id: 13640203, max_allowed_instances: 5, any_user: true, registered_users: []}, {id: 12147054, max_allowed_instances: 1, any_user: true, registered_users: []}, {id: 13639843, max_allowed_instances: 1, any_user: true, registered_users: []}]"
 
+- name: TENNANT_USERNAME
```

Comment:
This parameter and the newly added parameter are not used yet. 
let's keep it in mind that we'll need to set the corresponding CLI flags using these values.
Just like: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/0f67edf61d3aaf52444d19604f8425a1c288e63d/templates/service-template.yml#L928-L930